### PR TITLE
chore: add GitHub Actions deployment routine for component explorer

### PR DESCRIPTION
**Pull request by awentzel**
# Description
Implements continuous deployment on every push into master for the component explorer staging site on Azure.

## Motivation & context
Migrate explorer pipeline from ADO to GitHub Actions for greater transparency.

## Issue type checklist

- [x] **Chore**: A change that does not impact distributed packages.
- [ ] **Bug fix**: A change that fixes an issue, link to the issue above.
- [ ] **New feature**: A change that adds functionality.

**Is this a breaking change?**
- [ ] This change causes current functionality to break.

## Process & policy checklist

- [ ] I have added tests for my changes.
- [ ] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast-dna/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/en/contributing/standards) for this project.

### CODE REVIEW

File: .github/workflows/ci-explore-alpha.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
+name: CI Component Explorer
+
+on:
+  pull_request:
```
Comment (awentzel) [inline .github/workflows/ci-explore-alpha.yml]: `pull_request` will be changed to `push` upon successful validation testing.

File: .github/workflows/ci-explore-alpha.yml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    - master
+
+jobs:
+  build_linux:
+    runs-on: ubuntu-latest
+
+    env:
+      AZURE_WEBAPP_NAME: 'fast-explore'
+      AZURE_WEBAPP_PACKAGE_PATH: './packages/fast-component-explorer/www/'
+      AZURE_WEBAPP_SLOT_NAME: 'alpha'
```
Comment (awentzel) [inline .github/workflows/ci-explore-alpha.yml]: `alpha` will be replaced with `stage` upon successful validation

File: .github/workflows/ci-explore-alpha.yml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+        yarn build:app
+
+    - name: 'Deploy to Azure WebApp'
+      uses: azure/webapps-deploy@v2
+      with:
+        app-name: ${{ env.AZURE_WEBAPP_NAME }}
+        # publish-profile: ${{ secrets.AZURE_WEBAPP_PUBLISH_PROFILE_EXPLORE_FAST_DESIGN_ALPHA }}
+        package: ${{ env.AZURE_WEBAPP_PACKAGE_PATH }}
+        slot-name: env.AZURE_WEBAPP_SLOT_NAME
+        # startup-command: 'pm2 start /home/site/wwwroot/server.js --no-daemon'
```
Comment (awentzel) [inline .github/workflows/ci-explore-alpha.yml]: comments will be removed upon final validation

File: .github/workflows/cd-explore-stage.yml [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+    - name: Checkout Branch
+      uses: actions/checkout@master
+
+    - name: Install Lerna
+      run: sudo yarn global add lerna
+
+    - name: Install package dependencies
+      run: yarn install --frozen-lockfile
+
+    - name: Prepare Yarn workspaces
```
Comment (nicholasrice) [inline .github/workflows/cd-explore-stage.yml]: I *believe* yarn will actually prepare on install so this `lerna run prepare` step might not be necessary

Comment (chrisdholt) [reply .github/workflows/cd-explore-stage.yml]: Yeah I think this may be accurate, especially if done in the root.

**Pull request merged**